yoti_python_sdk/dynamic_sharing_service/extension/location_constraint_extension_builder.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+
+class LocationConstraintExtensionBuilder(object):
```

Comment:
Should this extend the ExtensionBuilder rather than being its own standalone class?

```suggestion
from .extension_builder import ExtensionBuilder

class LocationConstraintExtensionBuilder(ExtensionBuilder):
```